content/scripts/build.js

```diff
@@ -978,63 +1117,39 @@ class Builder {
     return locales;
   }
 
-  excludeSlug(metadata) {
+  excludeSlug(url) {
     // XXX would it be faster to compute a regex in the constructor
     // and use it repeatedly here instead?
-    const { slugsearch } = this.options;
-    if (slugsearch.length) {
-      const { mdn_url } = metadata;
-      return !slugsearch.some((search) => mdn_url.includes(search));
+    if (url) {
```

Comment:
Why would it NOT be true? Seems like a severe thing.

Reply:
👍 I'll remove that.